scripts/core/program.yml

```
@@ -264,6 +310,8 @@ desc: "Set a Program object specialization constant to a specific value"
 class: $xProgram
 name: SetSpecializationConstant
 ordinal: "0"
+analogue: "**clSetProgramSpecializationConstant**"
+details: "`hProgram` must not be an executable program, or a program created from a device binary."
```

probably works but best to be consistent and make this an array of strings.

```suggestion
details:
    - "`hProgram` must not be an executable program, or a program created from a device binary."
```